RadixWallet/Features/SplashFeature/Splash+View.swift

```diff
@@ -101,3 +77,36 @@ extension Splash.State {
 	public static let previewValue = Self()
 }
 #endif
+
+// MARK: - SplashView
+public struct SplashView: View {
+	var biometricsCheckFailed: Bool = false
+
+	public var body: some View {
+		ForceFullScreen {
```

Comment:
ForceFullScreen is an old hack that should not be needed or used. In fact it would be nice to get rid of it altogether, it's only used here an in one other spot (where it is even more dodgy). Check if it's not enough to do something like `.frame(maxHeight: .infinity)` instead. And for the other use site, maybe just replacing it with `Color.clear` will be enough.

Reply:
removed ForceFullScreen